includes/Settings.php

```diff
@@ -1008,6 +1012,182 @@ public function move_setting_after_id( $settings, $insert_settings, $after_setti
 		return $new_settings;
 	}
 
+	public function update_documents_settings(): void {
```

Comment:
Consider including `sections` in the function name to make it clearer that you're modifying specific sections rather than the overall document settings.